Docker/itk-base/Dockerfile

```diff
@@ -16,7 +17,11 @@ RUN curl -O https://cmake.org/files/v3.4/cmake-3.4.3.tar.gz && \
     cd /tmp/cmake-build && \
     ../cmake-3.4.3/bootstrap && \
     make -j$(nproc) && \
-    ./bin/cmake -DCMAKE_BUILD_TYPE:STRING=Release . && \
+    cd .. && \
+    mkdir /tmp/cmake-ssl-build && \
+    cd /tmp/cmake-ssl-build && \
+    ../cmake-build/bin/cmake -DCMAKE_USE_OPENSSL:BOOL=ON \
+    -DCMAKE_BUILD_TYPE:STRING=Release ../cmake-3.4.3 && \
```

Comment:
Great. I didn't know I good pass options from the bootstrap compilation of CMake. I tried first to build bootstrap-cmake and then build CMake with OpenSSL support in the same folder but I was running into errors. I'll try your strategy. Thanks!